client/driver.go

```diff
@@ -58,12 +59,22 @@ func Init(config config.Config) error {
 
 	ctx, cancel := util.GetContext(context.Background())
 	defer cancel()
-
-	drv, err := driver.NewDriver(ctx, &cconfig.Driver{
-		URL:               url,
-		ApplicationId:     config.ApplicationID,
-		ApplicationSecret: config.ApplicationSecret,
-	})
+	var drv driver.Driver
+	var err error
+	if config.UseTls {
```

Comment:
UseTLS

Reply:
Actually can we call it NoTLS and have TLS enabled by default?

Reply:
can we call it `USE_PLAINTEXT` instead and reverse the situation?

Reply:
discussed it offline and decided to move forward here with plaintext as default.